bundles/org.openhab.binding.octopusenergy/README.md

```diff
@@ -0,0 +1,167 @@
+# Octopus Energy Binding
+
+This binding allows OpenHAB to communicate with the public API from Octopus Energy (https://octopus.energy), an energy provider in the UK.
```

Comment:
```suggestion
This binding allows openHAB to communicate with the public API from Octopus Energy (https://octopus.energy), an energy provider in the UK.
```

Please change all cases to use lowercase o as that is the official way to spell openHAB.

Reply:
done